docker-compose.yaml

```diff
@@ -20,17 +21,24 @@ services:
       SCHEDULE_QUEUE_ADDRESS : schedule
       SCHEDULE_QUEUE_USER : artemis
       SCHEDULE_QUEUE_PASSWORD : artemis
+    networks:
+      - ffc-demo-claim
     volumes:
       - node_modules:/usr/src/app/node_modules
       - .:/usr/src/app
 
   ffc-demo-claim-postgres:
-    container_name: ffc-demo-claim-postgres${BUILD_NUMBER:-local}
+    container_name: ffc-demo-claim-postgres-${BUILD_NUMBER:-local}
     image: postgres:11.4-alpine
     environment:
       POSTGRES_DB: ffc_demo_claims
       POSTGRES_PASSWORD: ppp
       POSTGRES_USERNAME: postgres
+    networks:
+      - ffc-demo-claim
+
+networks:
+  ffc-demo-claim: {}

```

Comment:
Networks returned to base file as they are needed when running with `docker-compose.link.yaml` as well as with `docker-compose.override.yaml`. It is not necessary for tests but has no negative impact - docker-compose creates a network either way, it's just the name that changes.

Reply:
I think the network name should also include the build number as running two tests simultaneously (i.e. different PRs for the same repo) may fail as the network name will be the same between the two test runs.

Reply:
It doesn't seem possible to create a network with a variable name via docker-compose. I think we'll have the same issue if we don't specify the network name? Running tests twice in parallel, I see only one network created and one postgres instance either way.

Reply:
This might be a non issue as the full network name seems to be a combination of the project name and the network name, which would prevent conflicts on the build server

Reply:
Ah yes, I hadn't spotted that the build number is used in the `-p` flag for running tests from the pipeline.